packages/react-focus/src/components/FocusZone/FocusZone.types.ts

```diff
@@ -201,6 +201,20 @@ export interface IFocusZoneProps extends React.HTMLAttributes<HTMLElement> {
    */
   shouldRaiseClicks?: boolean;
```

Comment:
Are the new props intended to eventually replace this one? If so it should be marked as `@deprecated` with a message to use the new props instead.

Reply:
I'm iffy on whether we should deprecate it, since I could see how it would be kinda annoying to need to set both other props to `false` in order to not raise any clicks if that were the intent. I don't really know how this prop is used by consumers in practice though :P

Reply:
@ecraig12345 I wanted to introduce this as a non-breaking change for all consumers. It doesn't really deprecate `shouldRaiseClicks` but it could be used instead of it.

Think of it like audio setting where you can mute all sounds, or you can selectively mute just ambience/voice/music/... but the master setting is still beneficial to have.

Reply:
Hmm that makes sense. I wonder if it would make sense/be any better to combine them somehow, like `shouldRaiseClicks?: boolean | 'enter' | 'space'` maybe? Those are weird to add to a boolean prop name but it would reduce the added API surface.

Reply:
@ecraig12345 That wouldn't work for future-proofing, if we want both <kbd>Enter</kbd> and <kbd>Space</kbd> we would pass in true right now, but the moment we needed to extend this with another key it would break apart.

Ideally this would have been made into an `Enum` from start which would allow us to do composites like:
```ts
enum Keys {
  Space = 1 << 0,
  Enter = 1 << 1,
  Tab = 1 << 2 // or whatever
}

// ...
raiseClicksFor?: Keys
// ...

<FocusZone
  raiseClicksFor={Keys.Space | Keys.Enter}
  // ...
/>
```

I, however, didn't want to introduce such API, especially not overload a boolean property with an Enum so I opted for explicit booleans instead and hopefully we can make a more optimal solution in v9.